src/Nethermind/Nethermind.JsonRpc/Modules/Parity/ParityRpcModule.cs

```diff
@@ -67,8 +67,9 @@ public class ParityRpcModule : IParityRpcModule
             _peerManager = peerManager ?? throw new ArgumentNullException(nameof(peerManager));
         }
 
-        public ResultWrapper<ParityTransaction[]> parity_pendingTransactions()
-            => ResultWrapper<ParityTransaction[]>.Success(_txPool.GetPendingTransactions().Where(pt => pt.SenderAddress != null)
+        public ResultWrapper<ParityTransaction[]> parity_pendingTransactions(Address? address)
+            => ResultWrapper<ParityTransaction[]>.Success(_txPool.GetPendingTransactions()
```

Comment:
GetPendingTransactions returns a big copy of the TxPool, would be better to pass address to it and return only bucket for that address.